v2/manager.go

```diff
@@ -643,7 +666,14 @@ func (c *Manager) waitForEvents(ec chan<- Event, errCh chan<- error) {
 				}
 				ec <- e
 			} else {
-				errCh <- err
```

Comment:
Likely out of scope, but it seems flipping `if err := readKVStatsFile(c.path, "memory.events", out); err == nil {` logic would allow to save bunch of indentation and make code much more readable and with happy path left-aligned.